### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by condef5**
Feature #14 : Layout App 

This pull request include the following:

* Markup html
* Styles for app structure 
* Content structuring

### CODE REVIEW

File: public/channel.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+<body class="layout">
+  <div class="user">
+    <img class="user__image" src="https://dummyimage.com/35X35/ddd/ddd.jpg" alt="name-user">
+    <h4 class="user__name">Diego Cuevas</h4>
+  </div>
+  <div class="channel-header">#general</div>
+  <div class="channels">
+    <ul id="listChannel">
+      <li><a href="#"># general</a></li>
```
Comment (byteconde) [inline public/channel.html]: Good channels, some names are familiar for me 👀

File: public/channel.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <form id="newMessage" class="new-message">
+      <input type="text" name="message" placeholder="Type a Message...">
+      <button type="submit" class="submit">
+        <svg class="icon">
+          <use xlink:href="#icon-send">
+        </svg>
+      </button>
+    </form>
+  </div>
+  <svg class="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (byteconde) [inline public/channel.html]: SVGs are a good option when you need to work with HQ content. Actually we're going to work with vectors. Well done Frank

File: public/channel.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <defs>
+      <symbol id="icon-plus" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor"
```
Comment (DiegoCuevas) [inline public/channel.html]: why are you using ```stroke``?

Comment (condef5) [reply public/channel.html]: I use attribute `stroke` for give it color an svg, in this case I pass the current color of the svg through the value `currentColor`.

File: public/channel.html (hunk 2 above)
Comment (condef5) [reply public/channel.html]: Thanks bro

**Pull request merged**